htmd/ffevaluation/ffevaluate.py

```diff
@@ -256,9 +258,11 @@ def ffevaluate(mol, prm, betweensets=None, cutoff=0, rfa=False, solventDielectri
     >>> prm = parmed.amber.AmberParameterSet.from_structure(prmtop)
     >>> energies, forces, atmnrg = ffevaluate(mol, prm, betweensets=('resname SOD', 'water'))
     """
+    assert mol.box.shape[0] == 3 and mol.box.shape[1] == mol.coords.shape[2], 'Box dimensions have to be (3, numFrames), your Molecule has box of shape {}'.format(mol.box.shape)
+
```

Comment:
This should throw `ValueError` exception, rather than assert. As the error can be triggered by a user directly passing a wrong `box`.